src/CommonSpaces/CommonSpaces.jl

```diff
@@ -335,14 +354,17 @@ space = SliceXZSpace(;
     z_max = 1,
     periodic_x = false,
     n_quad_points = 4,
-    x_elem = 4,
-    staggering = Grids.CellCenter()
+    x_elem = 4
 )
 ```
 """
 function SliceXZSpace end
 SliceXZSpace(; kwargs...) = SliceXZSpace(Float64; kwargs...)
-SliceXZSpace(::Type{FT}; staggering::Staggering, kwargs...) where {FT} =
+SliceXZSpace(
+    ::Type{FT};
+    staggering::Staggering = Grids.CellCenter(),
+    kwargs...,
+) where {FT} =
```

Comment:
`staggering` should not have a default value, just like `z_elem` and alike. Users need to be aware of what they're making.